### PR TITLE
gitignore improvement

### PR DESCRIPTION
**Pull request by michal367**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -250,4 +250,21 @@ dmypy.json
 # Cython debug symbols
 cython_debug/
 
-# End of https://www.toptal.com/developers/gitignore/api/python,pycharm
\ No newline at end of file
+# End of https://www.toptal.com/developers/gitignore/api/python,pycharm
+
+
+# IDEs
+.idea
```
Comment (grzanka) [inline .gitignore:257]: .idea is already above

File: .gitignore [hunk 2]
```diff
@@ -250,4 +250,21 @@ dmypy.json
...
+.vscode
```
Comment (grzanka) [inline .gitignore:258]: Better to copy paste visual studio code example
https://www.toptal.com/developers/gitignore/api/visualstudiocode

File: .gitignore [hunk 3]
```diff
@@ -250,4 +250,21 @@ dmypy.json
...
+.settings
+.project
```
Comment (grzanka) [inline .gitignore:260]: What is using `.project` and `.settings` ?

File: .gitignore [hunk 4]
```diff
@@ -250,4 +250,21 @@ dmypy.json
...
+
+# virtualenvs
+venv*
+
+# pyinstaller output
+build
```
Comment (grzanka) [inline .gitignore:266]: `build` is already included

**Review by grzanka — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (michal367) [reply .gitignore:257]: folder .idea isn't above. Above are only some files inside it. Before I added this `git status` was showing me this folder.

File: .gitignore (hunk 3 above)
Comment (michal367) [reply .gitignore:260]: eclipse

Comment (grzanka) [reply .gitignore:260]: Then use this one: https://www.toptal.com/developers/gitignore/api/eclipse
image

File: .gitignore [hunk 5]
```diff
@@ -250,4 +250,21 @@ dmypy.json
...
+# End of https://www.toptal.com/developers/gitignore/api/python,pycharm
+
+
+# IDEs
+.idea
+.vscode
+.settings
+.project
+
+# virtualenvs
```
[the remark this answers is not in the document]
Comment (grzanka) [reply .gitignore:276]: OK, then add a comment explaining the reasons

File: .gitignore (hunk 2 above)
Comment (grzanka) [reply .gitignore:274]: nice that you added an entry, but please remove `.vscode` as well

**Review by grzanka — APPROVED** [review]
Looks good

**Pull request merged**